Frameworks/Foundation/NSMutableArray.mm

```diff
@@ -557,11 +557,33 @@ - (void)replaceObjectsInRange:(NSRange)aRange withObjectsFromArray:(NSArray*)oth
 }
 
 /**
- @Status Caveat
- @Notes Options are currently ignored
+ @Status Interoperable
+ @Notes NSSortConcurrent is ignored, but this is compatible with reference documentation
 */
 - (void)sortWithOptions:(NSSortOptions)opts usingComparator:(NSComparator)cmptr {
-    [self sortUsingFunction:CFNSBlockCompare context:cmptr];
+    if (opts | NSSortStable) {
```

Comment:
got me again.. nice catch

---
In reply to: [72351654](https://github.com/Microsoft/WinObjC/pull/673#discussion_r72351654) [](ancestors = 72351654)